rudder-webapp/SOURCES/Makefile

```diff
@@ -57,11 +57,15 @@ localdepends: ./rudder-sources ./maven/bin/mvn ./rudder-users.xml ./rudder-doc .
 ./rudder-users.xml: ./rudder-sources
 	cp ./rudder-sources/rudder/rudder-web/src/main/resources/demo-rudder-users.xml ./rudder-users.xml
 
+./.gitignore: ./configuration-repository-gitignore
+	cp ./configuration-repository-gitignore ./.gitignore
```

Comment:
I wanted to avoid committing a source file with a name beginning with a ```.```. I can see so many cases where this could cause trouble, because shells usually exclude these files from default operations (ls, cp, mv, ...). Also, I'm concerned how git would behave if it finds a ```.gitignore``` file, which is a special name for git, in a directory under git version control.